recipes/gvcftools/build.sh

```diff
@@ -1,5 +1,5 @@
 sed -i.bak 's^TABIX_LIB_DIR)$^TABIX_LIB_DIR) -L$(PREFIX)/lib^' src/Makefile
 sed -i.bak 's^TABIX_INCLUDE_DIR)$^TABIX_INCLUDE_DIR) -L$(PREFIX)/include^' src/Makefile
-sed -i.bak 's@^CFLAGS =@CFLAGS = -I$(PREFIX)/include@' src/redist/tabix/Makefile
+sed -r "s^tabix: (.*)^tabix: \1\n\tsed -i.bak '@CFLAGS =@CFLAGS = -I$\(PREFIX\)/include@ tabix/Makefile'^" redist/Makefile
```

Comment:
maybe a patch would be better if it gets to complicated? Conda supports patches natively, so it will apply them for you, if you specify the patch file in your meta.yaml